src/Sentry/Threading/ScopedCountdownLock.cs

```diff
@@ -139,7 +146,7 @@ internal LockScope(ScopedCountdownLock lockObj)
         internal void Wait()
         {
             var lockObj = _lockObj;
-            lockObj?._event.Wait();
+            lockObj?._event.Wait(Timeout.Infinite, CancellationToken.None);
```

Comment:
note: this is not a behavioral change, just making the behavior more apparent by "lifting" the method call of the parameterless `Wait` method